### DISCUSSION TITLE
Support for multi-step forms

### DISCUSSION THREAD

**Discussion by migueloller** [edmundhung/conform#691] [Ideas]
Hi there! 👋🏻 

I love Conform! It's made it extremely easy to set up progressively enhanced forms that are accessible and work well.

I've got an idea that's something more of a question. When building multi-step forms, is there a recommended approach with Conform? Could there be specific APIs that could make building multi-step forms easier? If one were building a multi-step form that can work without JS, what would that look like? Is there a good solution out there for progressively enhanced multi-step forms?

Would love to hear the community's thoughts on this and if this is something people would event want.

Cheers!

**Comment by bdrtsky** [comment]
Also wonder about the best approach of building multi-step forms with Conform. Makes me wonder if it's a right tool? Any examples would be highly appreciated.

**Comment by kenteramin** [comment]
I think a multi-step form is better thought of as several forms submitting parts of the required data.
You'd need some kind of a support for that from your BE. For example to have a draft stage for your entity.
Then you can locate your forms on separate pages, like `/companies/{id}/step-1`, `/companies/{id}/step-2`, etc
